CHANGELOG.md

```diff
@@ -5,40 +5,56 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#45)
+- Removed unnecessary `Unit` argument in creation functions for `HTMLImageElement` and `HTMLAudioElement` (#46)
 
 New features:
+- Added `HTMLHyperlinkElementUtils` module (#44)
+- Added `create` and `create'` functions for `Audio` (#32)
+- Added missing attributes for `HTMLImage`, including `srcset`, `currentSrc`, `sizes`, `referrerPolicy`, `decoding`, `loading` (#40)
+- Added common `PropName`, `AttrName`, and `ClassName` which can be reused across libraries representing properties, attributes, and CSS classes (#30)
 
 Bugfixes:
+- Fixed return type of `scrollX` and `scrollY` to be `Effect Number`, as these can return subpixel values (#17)
+- Removed `Newtype` instances for types with hidden constructors (#39)
+- Changed `HTMLInputElement.accept` attribute type to `String` (#47) 
 
 Other improvements:
+- Migrated CI to GitHub Actions and updated installation instructions to use Spago (#41)
+- Removed primes from foreign modules exports in preparation for PureScript 0.15 (#24)
+- Removed `return {}` from FFI implementations for a small performance improvement (#35)
+- Removed unused explicit exports (#39)
+- Updated HTML spec link (#34)
+- Added a CHANGELOG.md file and new pull request template (#48, #50)
+- Removed old pull request template (#49)
```

Comment:
Can #49 be merged into the 'added a changelog file and new pull request template' line?